source/base/mpi.cc

```diff
@@ -812,7 +814,890 @@ namespace Utilities
 #endif
     }
 
+    template <typename T1, typename T2>
+    void
+    ConsensusAlgorithmProcess<T1, T2>::process_request(
+      const unsigned int     other_rank,
+      const std::vector<T1> &buffer_recv,
+      std::vector<T2> &      request_buffer)
+    {
+      (void)other_rank;
+      (void)buffer_recv;
+      (void)request_buffer;
```

Comment:
Since this is implementation only, I would suggest you simply skip the `other_rank`, `buffer_recv`, and `request_buffer` names in the list of function arguments, then you do not have to use `(void)`. (Here and in a few function below.)